extension/_locales/en/messages.json

```diff
@@ -0,0 +1,82 @@
+{
+    "extensionName": {
+      "message": "Firefox Translations",
+      "description": "Name of the extension"
+    },
+    "extensionDescription": {
+      "message": "Translate websites in your browser without using the cloud.",
+      "description": "Description of the extension."
+    },
+    "loadingTranslationEngine": {
+      "message": "Loading Translation Engine",
+      "description": "Tells the user the translation engine is being loaded."
+    },
+    "translationLoadedWithErrors": {
+      "message": "Automatic translation is enabled but we found errors.",
+      "description": "Tells the user the translation engine was loaded but errors were found."
+    },
+    "checksumErrorsDownloadingEngine": {
+      "message": "We found checksum errors when downloading the translation engine.",
+      "description": "Tells the user there were errors downloading the translation engine."
+    },
+    "notfoundErrorsDownloadingEngine": {
+      "message": "We could not find the translation engine to download.",
+      "description": "Tells the user the translation engine could not be found to download."
+    },
+    "nodataDownloadingEngine": {
+      "message": "We found no data when downloading the translation engine.",
+      "description": "Tells the user the translation engine had not data when downloading."
+    },
+    "translationEnabled": {
+      "message": "Automatic Translation enabled.",
+      "description": "Tells the user the automatic translation was enabled."
+    },
+    "translationEnabledNoOT": {
+      "message": "Automatic Translation enabled but translation of forms is not supported for this language.",
+      "description": "Tells the user the automatic translation was enabled but form translations is not supported."
+    },
+    "timeoutDownloadingEngine": {
+      "message": "Timeout downloading the translation engine.",
+      "description": "Tells the user the download of the translation engine timed out."
+    },
+    "errorLoadingWasm": {
+      "message": "We found an error loading the translation engine.",
+      "description": "Tells the user we found and error loading the translation engine."
+    },
+    "statsMessage": {
+      "message": "Translation enabled. Words translated-per-second: $WORDSPERSECOND$.",
```

Comment:
Why the hyphens?

Reply:
Will remove them.

Reply:
To be clear, I'm asking as a non native speaker who finds this strange, not saying that's 100% wrong ;-)

Localization doesn't normally do content review for English, I just want to make sure we don't have to iterate over this in the future (and have to retranslate it as a consequence).

Reply:
Got it. We don't need hyphens here.